src/Components/Modal/Modal.tsx

```diff
@@ -16,16 +17,20 @@ const ModalContainer = styled.div`
   width: 440px;
   border-radius: 4px;
   padding: 20px 40px;
+  box-shadow: 0 4px 15px rgba(0, 0, 0, 0.15);
 `
 
-const Overlay = styled.div`
+const Overlay = styled.div.attrs<{ show?: boolean }>({})`
   position: fixed;
   width: 100%;
   height: 100%;
-  top: 0px;
-  left: 0px;
+  top: 0;
+  left: 0;
   z-index: 9998;
-  background: rgba(0, 0, 0, 0.3);
+  background: rgba(200, 200, 200, 0.5);
+  -webkit-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
```

Comment:
@l2succes - I just tried out the modal and noticed that these CSS blur props aren't applying to the page background -- anything I need to do?